source/common/common/to_lower_table.cc

```diff
@@ -0,0 +1,16 @@
+#include "common/common/to_lower_table.h"
+
+ToLowerTable::ToLowerTable() {
```

Comment:
The generation logic can be executed at compile time as a constexpr.  If you do this then the table will just become part of the binary rather than needing to be calculated at runtime.  Since this is just calculated once at startup the benefit is probably negligible.